test/Microsoft.TemplateEngine.Edge.UnitTests/NuGetApiPackageManagerTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Edge.Installers.NuGet;
+using Microsoft.TemplateEngine.TestHelper;
+using Xunit;
+
+namespace Microsoft.TemplateEngine.Edge.UnitTests
+{
+    public class NuGetApiPackageManagerTests
+    {
+        [Fact]
+        public async Task DownloadPackage_Latest()
+        {
+            string installPath = TestUtils.CreateTemporaryFolder();
+            EnvironmentSettingsHelper helper = new EnvironmentSettingsHelper();
```

Comment:
`helper`, needs to be disposed(to  delete folders used), also use `helper.CreateTemporaryFolder()` and `helper` will delete folders. Easiest way is to just do
```
EnvironmentSettingsHelper helper = new EnvironmentSettingsHelper();
```

Reply:
added helper via IClassFixture<EnvironmentSettingsHelper>